pkg/controllers/resources/ingester/statefulset.go

```diff
@@ -124,6 +124,15 @@ func (r *Ingester) statefulSet() (runtime.Object, resources.Operation, error) {
 	if r.ingester.Spec.LogFormat != "" {
 		container.Args = append(container.Args, "--log.format="+r.ingester.Spec.LogFormat)
 	}
+
+	if r.ingester.Spec.OtlpEnableTargetInfo != nil && !*r.ingester.Spec.OtlpEnableTargetInfo {
+		container.Args = append(container.Args, "--no-receive.otlp-enable-target-info")
+	}
+	if len(r.ingester.Spec.OtlpResourceAttributes) > 0 {
+		container.Args = append(container.Args, "--receive.otlp-promote-resource-attributes")
+		container.Args = append(container.Args, r.ingester.Spec.OtlpResourceAttributes...)
+	}
```

Comment:
the args may be 
`--receive.otlp-promote-resource-attributes=<attr1>`, `--receive.otlp-promote-resource-attributes=<attr2>`..., 
or 
`--receive.otlp-promote-resource-attributes=<attr1>,<attr2>...`